packages/fxa-auth-server/scripts/paypal-processor.ts

```diff
@@ -2,21 +2,29 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 import program from 'commander';
-import { setupAuthDatabase } from 'fxa-shared/db';
-import { StatsD } from 'hot-shots';
-import Container from 'typedi';
 
-import error from '../lib/error';
-import { CurrencyHelper } from '../lib/payments/currencies';
+import Container from 'typedi';
 import { PayPalHelper } from '../lib/payments/paypal';
 import { PayPalClient } from '../lib/payments/paypal-client';
+
+import {
+  AdditionalSetupFn,
+  initShared,
+} from '../lib/payments/processing-tasks-shared';
 import { PaypalProcessor } from '../lib/payments/paypal-processor';
-import { StripeHelper } from '../lib/payments/stripe';
-import { configureSentry } from '../lib/sentry';
 
 const pckg = require('../package.json');
 const config = require('../config').getProperties();
 
+const additionalSetup: AdditionalSetupFn = ({ log }) => {
```

Comment:
It seems odd that we add code to end up with the same setup in the same locations, but pass around where its done. Why not just move this under the initialization?